Fix a few

hgouraud opened the pull request:
in templ.ml remove "Lang in env1\n" trace message
in mutil.ml remove warning message for aliases
in lexicon.txt add missing %t for oc
in welcome.txt, request.ml, gwd.ml, config.ml/mli, templ.ml, perso.ml, plugins/gwxjg/gwxjg_data.ml
  replace "nb_of_persons" by "nb_persons", add corresponding "nb_families"
in gwd.ml add `if !debug then `before warnings for .gwf files
in distribution/gwd.sh, add a missing "

hgouraud committed:
a few fixes (env1, marrida, aliases, nb_of_persons)

a2line commented:
Note that those are fixes for 4 critical errors found in the log of the demo on Tuxfamily (randomly found by various bots scanning the little Grimaldi database).

a2line committed:
rm duplicated _f/_w also catched by window.location.path

a2line committed:
rm singular entry on cousins.5.0, only case with it

a2line committed:
make -bd parameter not mandatory when using memory cache

a2line committed:
add a short_dates_text_notag subfunction without bdo tags

a2line committed:
menubar cleanup with buttons/permissions/icon/css fixes

Fix permissions for buttons of the individual dropdown menu.
Fix code indentation and add comments to various sections.
Show permalink buttons based on access permissions, remove 'ocz' parameter if null (hg).
Show Geneanet search button to visitors and fix its Font Awesome icon name (no aliases).
Switch to first_name_key and surname_key in the URLs of sosa_prev/sosa_next buttons.
Add dates_notag after names in the titles of 'sosa_prev/sosa_next' buttons.
Rework both Sosa 1 buttons, add missing translation (working condition to hide second button still pending #fixme!)
Add scroll overflow to p_mod dropdown menu (fix #1736) and adjust left offset to 10% instead of 16%.
Restore 'navbar-light' class to make hamburger visible again in low resolutions.

hgouraud committed:
handle default_sosa_ref
